.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -8,15 +8,26 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for taking your time to report this bug!
-        Make sure you are running the latest version before reporting.
-  - type: input
+        Thank you for taking your time to report this bug! You gotta make sure these before reporting...
```

Wording can be improved here
```suggestion
        Thank you for taking your time to report this bug! Before opening an issue, please make sure that:
```